.packit.yaml

```diff
@@ -109,32 +103,19 @@ jobs:
 
   - job: propose_downstream
     trigger: release
-    update_release: false
     dist_git_branches:
-      - fedora-all-aarch64
-      - fedora-all-ppc64le
-      - fedora-all-x86_64
-      - epel-all-aarch64
-      - epel-all-ppc64le
-      - epel-all-x86_64
+      - fedora-all
+      - epel-9
```

Comment:
I'm only listing `epel-9` here because there is only `epel9` branch in dist-git. Also, `epel-all` currently expands to `epel-8`, `epel-9` and `epel-10.0`, which is [problematic](https://github.com/packit/packit/issues/2438), so if you want to do releases for `epel10`, it is necessary (after requesting the branch in dist-git) to list it here explicitly as `epel-10` or `epel10`.